draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
Don't we need to generate this ID regardless of transceiver direction?

Specifically, in example 7.2, should Alice have no MSID in her recvonly answer for the video tracks?